changelog.md

```diff
@@ -30,14 +30,30 @@
     * The **Quartz.Util.DictionaryExtensions** type was removed.
     
     * The 'Get(TKey key)' method of **DirtyFlagMap<TKey,TValue>** has been removed. You can instead use the
-      this[TKey key] indexer or �TryGetValue(TKey key, out TValue value)` to obtain the value for a given key.
-
+      this[TKey key] indexer or `TryGetValue(TKey key, out TValue value)` to obtain the value for a given key.
+    
+    * (Logging): `LibLog` has been removed and replaced with `Microsoft.Logging.Abstractions` (#1480).
+      * Examples in the `Quartz.Examples` project on setting up both Microsoft.Logging and Serilog with Quarztz.Net
+        Previously in Quartz 3.2 you would use: `Quartz.Logging.LogContext.SetCurrentLogProvider(loggerFactory)` the call has now changed to:
+        `LogProvider.SetLogProvider(loggerFactory)` where loggerFactory is a `ILoggerFactory`
+      * Another approach to configure logging, when setting quartz configuration options via HostBuilder, configure logging with the IServiceCollectionQuartzConfigurator:
+        An example to wire up Serilog:
+      ```c#
+      services.AddQuartz(q =>
+        {
+      * IsFixedSize
```

Comment:
instead of long code samples, maybe we should just point people to check migration guide and have a shorter version of the breaking change here, like "Liblog replaced with MS Logging, see migration guide"?

Reply:
Good point. fixed.